### ISSUE TITLE
Cannot download posts by shortcode

### ISSUE THREAD

**Issue by yuyoiuyoiu** [instaloader/instaloader#2389]
**Describe the bug**
Cannot download posts by shortcode

**To Reproduce**
instaloader --load-cookies Firefox --no-metadata-json --abort-on=302,400,429,401 -- -XXXXXXXX

**Error messages and tracebacks**
Cookies loaded successfully from firefox
myaccount has been successfully logged in.
Next time use --login=myaccount to reuse the same session.

Download aborted: Query to https://www.instagram.com/graphql/query responded with "302 Found" redirect to https://www.instagram.com/.
Saved session to xxxxxxxxxx

HTTP redirect from https://www.instagram.com/graphql/query to https://www.instagram.com/
JSON Query to graphql/query: Expecting value: line 1 column 1 (char 0) [retrying; skip with ^C]

HTTP redirect from https://www.instagram.com/graphql/query to https://www.instagram.com/
JSON Query to graphql/query: Expecting value: line 1 column 1 (char 0) [retrying; skip with ^C]

HTTP redirect from https://www.instagram.com/graphql/query to https://www.instagram.com/
-XXXXXXXX: JSON Query to graphql/query: Expecting value: line 1 column 1 (char 0)
Saved session to xxxxxxxxxx

Errors or warnings occurred:
-XXXXXXXX: JSON Query to graphql/query: Expecting value: line 1 column 1 (char 0)

**Instaloader version**
4.13.1

**Comment by hlyylly** [comment]
'query_hash': '2b0673e0dc4580674a88d426fe00ea90',  has wrong

**Comment by stephenhky** [comment]
I think a number of issues these few days are due to some API changes: https://developers.facebook.com/docs/instagram-platform/changelog/

**Comment by DominikHracek** [comment]
That'd make sense. I also started having issues now and not earlier, and the changelog checks it. When downloading I get:

`instaloader --login=<username> -- -<post_id>
Loaded session from /home/user/.config/instaloader/session-<username>.
Logged in as <username>.
-<post_id>: 400 Bad Request - "fail" status, message "feedback_required" when accessing https://www.instagram.com/graphql/query?<query_hash>,<variable>,<short_code>
Saved session to /home/user/.config/instaloader/session-<username>.

Errors or warnings occurred:
-<post_id>: 400 Bad Request - "fail" status, message "feedback_required" when accessing https://www.instagram.com/graphql/query?<query_hash>,<variable>,<short_code>
`

**Comment by Peter058-a11y** [comment]
I cannot download the reels. Here is what I'm get, I'm end up successfully downloaded the profile picture only.

**instaloader --login=username profile [profile]**

Session is loaded and account is logged in.

**JSON Query to api/v1/users/web_profile_info/?username=profile: 404 Not Found when accessing https://i.instagram.com/api/v1/users/web_profile_info/?username=profile [retrying; skip with ^C]
JSON Query to api/v1/users/web_profile_info/?username=profile: 404 Not Found when accessing https://i.instagram.com/api/v1/users/web_profile_info/?username=profile [retrying; skip with ^C]
Profile profile does not exist.
The most similar profiles are: photo__.profile._, profile_pictures_rating_
Trying again anonymously, helps in case you are just blocked.

HTTP redirect from https://i.instagram.com/api/v1/users/web_profile_info/?username=profile to https://i.instagram.com/accounts/login/?next=/api/v1/users/web_profile_info/
Redirected to login page. Use --login or --load-cookies.
[1/1] Downloading profile [profile]
Retrieving posts from profile [profile].
Saved session to path.**

**Comment by Peter058-a11y** [comment]
> > Profile profile does not exist.
> > The most similar profiles are: photo__.profile._, profile_pictures_rating_
> > Trying again anonymously, helps in case you are just blocked.
> 
> It seems that you are downloading posts from an empty account? Maybe you should check the profile name again

Any account would prompt these output. Not an empty account for sure. I usually ignore such output cause previously i can download the posts and reels

**Comment by ChitranshS** [comment]
### On running this code:
```
import instaloader
L = instaloader.Instaloader()
post = instaloader.Post.from_shortcode(L.context, "C_ztYi6gvKZ")
```
### Getting this error:

JSON Query to graphql/query: 401 Unauthorized - "fail" status, message "Please wait a few minutes before you try again." when accessing https://www.instagram.com/graphql/query?query_hash=2b0673e0dc4580674a88d426fe00ea90&variables=%7B%22shortcode%22%3A%22C_ztYi6gvKZ%22%7D [retrying; skip with ^C]

**Comment by ChitranshS** [comment]
Found the new url for graphql
Can this be fixed?
https://www.instagram.com/graphql/query/?doc_id=17867389474812335&variables=%7B%22include_logged_out%22%3Atrue%2C%22include_reel%22%3Afalse%2C%22shortcode%22%3A%22C_ztYi6gvKZ%22%7D

**Comment by idjumatov** [comment]
I am having the same issue.
JSON Query to graphql/query: Expecting value: line 1 column 1 (char 0)

**Comment by aandergr** [comment]
Fixed in [Version 4.13.2](https://github.com/instaloader/instaloader/releases/tag/v4.13.2).

**Comment by mirh** [comment]
> instaloader considers the first profile in your command as an account named 'profile', so it shows that the account 'profile' does not exist

That is really NOT clear in all the documentation honestly.
